tests/test_samsa.py

```diff
@@ -15,64 +15,56 @@ def test_get_scenes():
 
 
 @pytest.mark.skip(reason='TODO: Test probably broken, need to investigate')
```

Comment:
Shoule we remove the skip?

Reply:
No, because some of the expected values (I don't remember where we got them from) are still different from the ones we manage to compute (which I put as comments). The next step now is to review the calculation itself and, perhaps, change the tests accordingly.